### DISCUSSION TITLE
英文模式下，区分右下角大小写的英文形态

### DISCUSSION THREAD

**Discussion by Zcb991** [rime/home#1655] [Ideas]
我使用的是小鹤双拼。
如果是小写英文，则左下角显示为小写的a；
如果按下了capslock（大写锁定键），则右下角显示为大写的A。
如下图所示：
![image](https://github.com/user-attachments/assets/ad63e9d7-3d0f-4ebc-8f25-4583607d634c)

**Comment by HBUzxl** [comment]
复议，总是区分不清楚当前状态是大写还是小写

**Comment by yfdyh000** [comment]
怀疑系统不支持区分。
https://github.com/rime/weasel/issues/361
https://github.com/rime/home/issues/1561
https://github.com/rime/squirrel/issues/764 （Mac，可能无关）
